test/language/expressions/postfix-decrement/arguments.js

```diff
@@ -2,11 +2,21 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-es5id: 11.3.2-2-1-s
-esid: sec-postfix-decrement-operator
+esid: sec-update-expressions
 description: >
-    Strict Mode - SyntaxError is thrown if the identifier 'arguments'
-    appear as a PostfixExpression(arguments--)
+  It is an early Syntax Error if AssignmentTargetType of LeftHandSideExpression is strict. (arguments)
```

Comment:
not sure if this should be unified to `test/language/expressions/postfix-decrement/arguments-nostrict.js`, are the semantics there only applying to no strict?

Reply:
same for `test/language/expressions/postfix-decrement/eval.js`

Reply:
and the other postfix/prefix-increment equivalents

Reply:
Did you see my last comment here: https://github.com/tc39/test262/pull/1642#issuecomment-448316186? There are explicit semantics for strict mode and non-strict mode, and those semantics are different.

Reply:
Thanks for the clarification, I missed that notification.